src/Widgets.cpp

```diff
@@ -1,6 +1,6 @@
 #include "Widgets.hpp"
-#include "window.hpp" // WINDOW_MOD_CTRL
-#include <GLFW/glfw3.h> // key codes
+//#include "window.hpp" // WINDOW_MOD_CTRL
```

Comment:
Apparently `WINDOW_MOD_CTRL` and `GLFW/glfw3` are included somewhere else, do you know where? But anyway, we can remove these commented out lines entirely.

Reply:
When someone asked about window.hpp on the forum the other day, Andrew replied:

"Including any Rack header file other than rack.hpp is unsupported/unstable. rack.hpp includes all of Rack’s plugin API"

So yes, safe to remove these entirely.